app/helpers/ternary.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+/**
+ * Returns alt if subject is falsey.
+ */
+export default Ember.Helper.helper(function([ subject, alt ]) {
```

Comment:
I did see that `{{if ... }}` could take three args and use the first arg twice in this case. I think I'll go that route.